ecosystem/sep-0038.md

```diff
@@ -174,13 +176,18 @@ This endpoint can be used to fetch the [indicative](https://www.investopedia.com
 
 These prices are indicative. The actual price will be calculated at conversion time once the Anchor receives the funds from a User.
 
-The prices returned include any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, or other factors.
+The prices returned include any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, amount, delivery method, country code, or other factors.
 
 #### Request
 
 Name | Type | Description
 -----|------|------------
 `sell_asset` | string | The asset you want to sell, using the [Asset Identification Format](#asset-identification-format).
+`sell_amount` | string | The amount of `sell_asset` the client would exchange for each of the `buy_assets`.
+`sep` | string | The name of the SEP that will be executing the quoted exchange. As of now, the ones that can make use of RFQ are `SEP-6` or `SEP-31`.
```

Comment:
I'm hesitant for us to include a SEP name in here. Can we include an `intent` or similar instead? For example, `"intent": "transfer"` for the transfer API SEPs SEP-6/24, `"intent": "cross_border_payment"` for SEP-31. There are probably better names.

Reply:
@JakeUrban wondering what your thoughts are on this.

Reply:
Regarding `"intent"` vs `"sep"`, I think both have downsides: in the future, we could end up having different protocols for "cross_border_payment" or different kinds of quote-related actions inside the same SEP, so both `&sep=6` or `&intent=cross_border_payment` could become insufficient to describe the intended action. TBH I like the "intent" key for it being more generic and easier to expand, but I also like the values "sep6" and "sep31" as they give a precise definition of what it'll be used for and they're also easier to expand with edge cases.

---

> `delivery_method` is only used for off-chain --> on-chain (i.e. deposit) flows, so we may need another attribute for the opposite flow, describing how funds are delivered to the user.

@JakeUrban: maybe we could tweak the description of delivery_methods a bit so it can be used for both buy_asset and sell_asset:
* Update the description of delivery_methods to say `An array of objects describing the methods a client can use to deliver funds to or withdraw funds from the anchor.`
* Update the `/quote`, `/price` and `/prices` request parameters to have `buy_delivery_method` and `sell_delivery_method` as parameters that should be used when buy_asset or sell_asset are non-Stellar assets.

Reply:
> Update the description of delivery_methods to say An array of objects describing the methods a client can use to deliver funds to or withdraw funds from the anchor.

I think this approach would make the spec more confusing and make it easier to pass semantically invalid `delivery_method` values. For example, a client could potentially pass `cash_pickup` for an off-chain --> on-chain transaction, when the appropriate value would've been `cash_dropoff`.

From a design perspective, the methods a client uses to deliver funds to the anchor are disjoint from the methods an anchor supports for delivering funds to the client/user. Why describe both sets of methods under the same object?

What if we changed the existing `delivery_methods` object & parameter to be `client_delivery_method` and added a `anchor_delivery_method` object & parameter? So `/quote`, `/price`, and `/prices` requests would include a `client_delivery_method` if it was an off-chain --> on-chain exchange and an `anchor_delivery_method` if it was an on-chain --> off-chain exchange.

Reply:
@JakeUrban: I applied your suggestion in https://github.com/stellar/stellar-protocol/pull/989/commits/533d52fa85965b55d87ec71b3388c0e0292278aa. 

Q: should this trigger a major version increment from `1.2.0` to `2.0.0` or are we ok going to 1.3.0?

Reply:
> Q: should this trigger a major version increment from 1.2.0 to 2.0.0 or are we ok going to 1.3.0?

Thats a good question. Technically this is a breaking change (renaming an existing field) but since the SEP is in a draft state I don't think we need to strictly adhere to semantic versioning. @leighmcculloch curious on your thoughts here.

Reply:
I'd try and keep the SEP version under 1 until it is in an Active state. We have no rules or process for this, but I've found that useful since v0 is a break anything you want version in regards to semantic versioning.